lib/matplotlib/backend_bases.py

```diff
@@ -175,20 +175,28 @@ def draw_if_interactive(cls):
                 cls.trigger_manager_draw(manager)
 
     @classmethod
-    def show(cls, block=None):
+    def show(cls, block=None, warn=True):
```

Comment:
I don't think we need the `warn` parameter. Client applications can use the filter mechanism of the warnings module, if they really want to suppress the warning.

Reply:
ok, I'll remove this parameter. Thanks.